conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalytics.java

```diff
@@ -88,6 +88,14 @@ public ConvivaAnalytics(BitmovinPlayer player,
                             String customerKey,
                             Context context,
                             ConvivaConfiguration config) {
+        this(player, customerKey, context, new ConvivaConfiguration(), null);
+    }
+
+    public ConvivaAnalytics(BitmovinPlayer player,
+                            String customerKey,
+                            Context context,
+                            ConvivaConfiguration config,
+                            Client client) {
```

Comment:
I see why you need this however I'm not very happy with this being a public initializer 🤔 

I'm not into Java but can we make this an internal initializer? (not exposed through the public API)

Reply:
Did not find solution to overcome this